main.go

```diff
@@ -56,12 +58,28 @@ const (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+
+	// @TODO read /var/run/secrets/kubernetes.io/serviceaccount/namespace to get
+	// the correct namespace if it's installed in k8s
+	operatorNamespace = "k4e-operator-system"
 )
 
 var Config struct {
+
+	// NOTE DEPRECATE
 	// The port of the HTTP server
 	HttpPort uint16 `envconfig:"HTTP_PORT" default:"8888"`
 
+	// The port of the HTTPs server
+	HttpsPort uint16 `envconfig:"HTTP_PORT" default:"8443"`
```

Comment:
It needs to be exposed in the service (https://github.com/jakub-dzon/k4e-operator/blob/e982e6d7a61045ccab22161cf48426da803953dc/config/manager/manager.yaml#L60).

Reply:
8443 is taken by by the other rbac proxy container
```
→ git grep 8443
config/default/manager_auth_proxy_patch.yaml:        - "--secure-listen-address=0.0.0.0:8443"
config/default/manager_auth_proxy_patch.yaml:        - containerPort: 8443
config/rbac/auth_proxy_service.yaml:    port: 8443

```